src/test/java/org/folio/rest/support/AdditionalHttpStatusCodes.java

```diff
@@ -1,5 +1,6 @@
 package org.folio.rest.support;
 
+//TODO remove it and use io.netty.handler.codec.http.HttpResponseStatus.UNPROCESSABLE_ENTITY
```

Comment:
I agree, this constant should possibly be removed. 

The RAML Module Builder defines it's own set of [HTTP status codes](https://github.com/folio-org/raml-module-builder/blob/a058d1c5be1d70d014fb404cff7deff9dffda5e4/util/src/main/java/org/folio/HttpStatus.java)

Why are you suggesting that we use the netty defined HTTP status code constants instead?

Reply:
Because 422 http status is called Unprocessable Entity but in RAML Module Builder set it's HTTP_VALIDATION_ERROR (with status code 422)

Reply:
> Why are you suggesting that we use the netty defined HTTP status code constants instead?

[in  HTTP status codes](https://github.com/folio-org/raml-module-builder/blob/a058d1c5be1d70d014fb404cff7deff9dffda5e4/util/src/main/java/org/folio/HttpStatus.java#L84) it's called HTTP_VALIDATION_ERROR(422) but in the world It's well known as 422 Unprocessable Entity
It was confused me

Reply:
> > Why are you suggesting that we use the netty defined HTTP status code constants instead?
> 
> [in HTTP status codes](https://github.com/folio-org/raml-module-builder/blob/a058d1c5be1d70d014fb404cff7deff9dffda5e4/util/src/main/java/org/folio/HttpStatus.java#L84) it's called HTTP_VALIDATION_ERROR(422) but in the world It's well known as 422 Unprocessable Entity
> It was confused me

Could it be preferably to raise a change for the RAML Module Builder to rename it, rather than create a need to introduce a new top level dependency purely for a single constant (that presumably is differently modelled).

Reply:
Here is the ticket https://issues.folio.org/browse/RMB-445

Reply:
In the meantime, please either remove or update the TODO to reflect the RAML Module Builder issue

Reply:
Done